internal/provider/physical_replication_stream_test.go

```diff
@@ -0,0 +1,229 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/cockroachdb/cockroach-cloud-sdk-go/v6/pkg/client"
+	mock_client "github.com/cockroachdb/terraform-provider-cockroach/mock"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+type clusterReqMatcher struct {
+	ExpectedName string
+}
+
+func (m clusterReqMatcher) Matches(x any) bool {
+	req, ok := x.(*client.CreateClusterRequest)
+	if !ok {
+		return false
+	}
+	return req.Name == m.ExpectedName
+}
+
+func (m clusterReqMatcher) String() string {
+	return fmt.Sprintf("clusterReqMatcher %s", m.ExpectedName)
+}
+
+func TestIntegrationPhysicalReplicationStreamResource(t *testing.T) {
```

Comment:
It's preferable to have both integration and acceptance tests for each feature.  Was your concern about the length or cost of spinning up 2 advanced clusters?

Reply:
I've added acceptance tests.